### PR TITLE
Fixed issue where changing the number of children caused nothing to display

### PR DESCRIPTION
**Pull request by thzinc**
There was an issue where changing length of children caused nothing to display because activeIndex was incrementing higher than children.length.

Thanks @sunyang713 for making this component available! If there's any changes you'd prefer me to make, let me know. I'd love to help get this version published soon, if you're available to do so.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-fadethrough",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (sunyang713) [inline package.json]: Don't change this, I'll do a standalone commit to update the version number and publish to npm.

File: dist/react-fadethrough.js [hunk 2]
```diff
@@ -150,12 +150,11 @@ var Fadethrough = function (_Component) {
...
-      var i = this.state.activeIndex + 1;
-      if (i === this.props.children.length) this.setState(function (prevState) {
-        return { activeIndex: 0 };
-      });else this.setState(function (prevState) {
-        return { activeIndex: i };
+      var i = (this.state.activeIndex + 1) % this.props.children.length;
+      this.setState(function (prevState) {
+        return { activeIndex: i || 0 };
       });
+      console.log(i);
```
Comment (sunyang713) [inline dist/react-fadethrough.js]: Delete console.log

File: src/Fadethrough.js [hunk 3]
```diff
@@ -28,11 +28,8 @@ export default class Fadethrough extends Component {
   }
 
   next() {
-    let i = this.state.activeIndex + 1
-    if (i === this.props.children.length)
-      this.setState(prevState => ({ activeIndex: 0 }))
-    else
-      this.setState(prevState => ({ activeIndex: i }))
+    const i = (this.state.activeIndex + 1) % this.props.children.length
+    this.setState(prevState => ({ activeIndex: i || 0 }))
```
Comment (sunyang713) [inline src/Fadethrough.js]: `i` should always be number, so it's not necessary to do `i || 0`

**Review by sunyang713 — CHANGES_REQUESTED** [review]
Hi, thanks so much for doing this! Using the `%` operator is definitely cleaner. One comment, and make sure to generate the distribution file using `npm run build` (it looks like you updated it manually; there's a console.log statement in there).

File: dist/react-fadethrough.js (hunk 2 above)
Comment (thzinc) [reply dist/react-fadethrough.js]: Whoops. Will do

File: package.json (hunk 1 above)
Comment (thzinc) [reply package.json]: Got it.

File: src/Fadethrough.js (hunk 3 above)
Comment (thzinc) [reply src/Fadethrough.js]: True. I'll update this.

**Comment by thzinc** [comment]
I've made the changes you requested. Thanks for the feedback!

**Review by sunyang713 — APPROVED** [review]

**Comment by sunyang713** [comment]
Awesome. Thanks!

**Pull request merged**